app/assets/stylesheets/finder_frontend.scss

```diff
@@ -399,9 +399,17 @@
   }
 }
 
-.govuk-label--inline {
+.app-label--inline {
```

Comment:
I wonder if this could be misleading - it looks very similar to our component convention of naming something with an `app-c-` prefix. Same for `app-select` below.